pages.fr/common/brew-bundle.md

```diff
@@ -0,0 +1,28 @@
+# brew bundle
+
+> Outil de gestion de paquets pour Homebrew, Homebrew Cask et le Mac App Store.
+> Plus d'informations : <https://github.com/Homebrew/homebrew-bundle>.
+
+- Installe tous les paquets listés dans le Brewfile situé dans le chemin courant :
+
+`brew bundle`
+
+- Installe tous les paquets listés dans le Brewfile situé au chemin spécifié :
+
+`brew bundle --file={{chemin/du/fichier}}`
```

Comment:
We should normalize mentions to path files as described in [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md). 
```suggestion
`brew bundle --file={{chemin/vers/fichier}}`
```